Code/Editor/Plugins/ProjectSettingsTool/PlatformSettings_Android.cpp

```diff
@@ -165,8 +165,8 @@ namespace ProjectSettingsTool
             if (editContext)
             {
                 editContext->Class<AndroidSplashscreens>("Splashscreens", "All splashscreen overrides for Android.")
-                    ->DataElement(0, &AndroidSplashscreens::m_landscapeSplashscreens)
-                    ->DataElement(0, &AndroidSplashscreens::m_portraitSplashscreens)
+                    ->DataElement(nullptr, &AndroidSplashscreens::m_landscapeSplashscreens)
```

Comment:
Actually sorry I just noticed I think this is wrong, this function is an overload and I think this should really be `AZ::Edit::UIHandlers::Default` not `nullptr` (see `EditContext.h` for the overloads)

Reply:
Yup, You're 100% that this is incorrect, will fix in ~1h

Reply:
Fixed in f2c482b

Reply:
Awesome thanks! 🙂